### PR TITLE
Myassignmentmp

### PR DESCRIPTION
**Pull request by mathieu-peh**
Good evening, 
Here my pull request for the lab 4 assignment. 
Best regards, 
Mathieu Péharpré

### CODE REVIEW

**Comment by mathurinm** [comment]
HI @mathieu-peh, you have a conflict so the tests don't run. Can you fix it ?

**Comment by mathieu-peh** [comment]
Good afternoon @mathurinm, I just resolved the conflict. Is it ok now ?

**Comment by mathurinm** [comment]
Yes it is ! I'll let you fix the remaining tests, ping me again when the CI is green :+1:

File: students.txt [hunk 1]
```diff
@@ -28,8 +28,8 @@ Steve,Moses
 Zofia,Smolen
 Elie,Cochard--Marchewka
 Guillaume,Plassais
-Mathieu,Péharpré
-Anastasia,Nesterova
+Anastasia,Nesterova X
```
Comment (mathurinm) [inline students.txt:31]: @mathieu-peh  can you not modify this line ?

File: numpy_questions.py [hunk 2]
```diff
@@ -64,4 +71,12 @@ def wallis_product(n_terms):
     """
     # XXX : The n_terms is an int that corresponds to the number of
     # terms in the product. For example 10000.
-    return 0.
+    pi = 1
+
+    if n_terms == 0:
```
Comment (mathurinm) [inline numpy_questions.py:76]: @mathieu-peh can you explain this line ?

File: students.txt (hunk 1 above)
Comment (mathieu-peh) [reply students.txt:31]: Should I remove the "X" ? Or exchange the order of the names, Anastasia then me ?

File: numpy_questions.py (hunk 2 above)
Comment (mathieu-peh) [reply numpy_questions.py:76]: Which line ? The "return 0" one ? I don't have this line in my original code

File: students.txt (hunk 1 above)
Comment (mathurinm) [reply students.txt:31]: In red are the lines before your PR, in green after. THe line with Anastasia's name should not be modified, hence appear neither in red, nor in green

File: numpy_questions.py (hunk 2 above)
Comment (mathurinm) [reply numpy_questions.py:76]: The github comment appears exactly under the line it refers to, so line 76 here

File: students.txt (hunk 1 above)
Comment (mathieu-peh) [reply students.txt]: Good morning, I tried to change it but then I get a message telling me that there is a conflict on the Student document at the line of Anastasia's

**Comment by mathurinm** [comment]
:+1:

**Pull request closed without merging**